software_subprojects.md

```diff
@@ -1,18 +1,20 @@
 # Software Subprojects
 
-Software design and development in Project Jupyter is organized into a set of Software Subprojects. These software Subprojects often map onto GitHub repositories/organizations, but that is not strictly required. The lifecycle of Software Subprojects is [described in detail here](newsubprojects.md).
+Software design and development in Project Jupyter is organized into a set of Software Subprojects. The lifecycle of Software Subprojects is described in detail [here](newsubprojects.md). A list of Subprojects is [here](./list_of_subprojects.md).
+
+Software Subprojects often carry out their work in GitHub repositories/organizations. Any Subproject repositories or organizations that are on GitHub must belong to the [Project Jupyter GitHub enterprise organization](https://github.com/enterprises/jupyter). This enables Project Jupyter to have consistent policies across all of Jupyter while enabling Subprojects to have autonomy in their GitHub orgs and repositories.
```

Comment:
Often -> Mostly, or remove altogether? I think in practice, at least as of today (and even more so with the new Enterprise), Github is officially Jupyter's software development home, and it should be our official central reference. It's not perfect, concerns about corporate control and data ownership remain, but it's the best we have in 2024 and the community uses it. So I suggest we make it clear that software work is officially done at Github.

This doesn't preclude using other specific tools for some tasks, but those are complementary and as needed, not the main home of software development.

Reply:
I updated the subproject document to put the GitHub requirement down in the list of subproject responsibilities.